### PR TITLE
Handle main

### PR DESCRIPTION
**Pull request by ZeyadHessien-Dev**
# Handle Main File 

 - remove all comments
 - build fake scaffold
 - this is a screenshot

Screenshot 2024-07-31 at 23 34 46

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -30,95 +30,7 @@ class MyApp extends StatelessWidget {
...
-    setState(() {
-      // This call to setState tells the Flutter framework that something has
-      // changed in this State, which causes it to rerun the build method below
-      // so that the display can reflect the updated values. If we changed
-      // _counter without calling setState(), then the build method would not be
-      // called again, and so nothing would appear to happen.
-      _counter++;
-    });
-  }
+      home: Scaffold(appBar: AppBar(title: const Text('Github Statgung'))));
```
Comment (ZEYAD-DEVELOP) [inline lib/main.dart:33]: nice line

**Review by ZEYAD-DEVELOP — COMMENTED** [review]
excellent

**Comment by ZeyadHessien-Dev** [comment]
but I see it's good

**Review by ZEYAD-DEVELOP — APPROVED** [review]

**Review by ZEYAD-DEVELOP — APPROVED** [review]

**Comment by ZeyadHessien-Dev** [comment]
ok

**Pull request merged**